src/app/components/mnoe-config/mnoe-admin-config.svc.coffee

```diff
@@ -97,5 +97,10 @@
     else
       $log.debug("DASHBOARD_CONFIG.marketplace.pricing.currency missing")
       'AUD'
+  @isProvisioningEnabled = () ->
+    if DASHBOARD_CONFIG.provisioning?.enabled?
+      DASHBOARD_CONFIG.provisioning.enabled
```

Comment:
This PR will need to be rebased/merged on this PR https://github.com/maestrano/mnoe-admin-panel/pull/71 as I reworked the feature flags in it.